content/applications/hr/lunch.rst

```diff
@@ -0,0 +1,305 @@
+
+   .. image:: lunch/extras.png
+      :align: center
+      :alt: The first of the extras configured for pizza toppings.
+
+   The pizzeria is offering a free beverage with any purchase. To set this up, the :guilabel:`Extra
+   2 Label` is set to `Beverages`, and the :guilabel:`Extra 1 Quantity` is set to :guilabel:`Only
+   One`. The various beverage choices are added, and the cost for each remains zero.
+
+   .. image:: lunch/beverages.png
+      :align: center
+      :alt: The second of the extras configured for a free beverage with purchase.
+
+Locations
+=========
+
+By default, Odoo creates an `HQ Office` location when the *Lunch* application is installed. If a
+company has more than one location, they must be configured.
+
+To add a location, navigate to :menuselection:`Lunch app --> Configuration: Locations`. The
+currently configured locations appear in a list view. Click the :guilabel:`New` button in the
+top-left corner and a blank line appears beneath the last location in the list.
+
+Enter the name of the location in the field. Next, click into the :guilabel:`Address` field to the
+right of the name, and enter the location's address. It is possible to enter multiple lines in the
+address field.
+
+Repeat this for all locations that need to be added.
+
+.. image:: lunch/locations.png
+   :align: center
+   :alt: A list view of the locations with the new button highlighted.
+
+Products
+========
+
+Before any orders can be placed, the individual products that are being offered must be configured,
+as no products are pre-configured in Odoo by default.
+
+First, navigate to the products page by going to :menuselection:`Lunch app --> Configuration:
+Products`. Next click the :guilabel:`New` button in the top-left corner and a blank product form
+loads.
+
+Enter the following information on the form:
+
+- :guilabel:`Product Name`: enter the name for the product. This field is **required**.
+- :guilabel:`Product Category`: using the drop-down menu, select the category this product falls
+  under. This field is **required**.
+- :guilabel:`Vendor`: using the drop-down menu, select the vendor that supplies this product. This
+  field is **required**.
+- :guilabel:`Price`: enter the price for the product. The currency is determined by the localization
+  of the company.
+- :guilabel:`Description`: enter a description of the product in this field. This description
+  appears beneath the product photo when users are viewing the options for the day.
+- :guilabel:`New Until`: using the calendar pop-over, select the date that the product will no
+  longer be labeled as new. Until this date, a green `New` tag appears on the product.
+- :guilabel:`Company`: if the product should only be available to a specific company, select it from
+  the drop-down menu. If this field is left blank, this product is available for all companies in
+  the database.
+- Image: hover over the image box in the top-right, and click the :guilabel:`✏️ (pencil)` edit icon
```

Comment:
I *think* location queues like "top-right" are compound adjectives and they require a noun to follow them in technical writing.
```suggestion
- Image: hover over the image box in the top-right corner of the form, and click the :guilabel:`✏️ (pencil)` edit icon
```